hw2/04-background.js

```diff
@@ -0,0 +1,27 @@
+let timeInterval = setInterval(function () {
+  let randomColor1 = Math.floor(Math.random() * 255);
+  document.body.style.backgroundColor = "#" + randomColor1;
+}, 3000);
+let btnSelector = document.querySelector(".btn");
+btnSelector.addEventListener("click", changeBackgroundColor);
+function changeBackgroundColor(event) {
```

Comment:
Please use named function expressions instead of function declarations. This is the preference of the AirBnB style guide 7.1 [read about it here](https://github.com/airbnb/javascript#functions--declarations)